### ISSUE TITLE
请教：采用自定义数据集分类训练，'rewards/format_reward': 基本都是0

### ISSUE THREAD

**Issue by rxmao** [Liuziyu77/Visual-RFT#217]
类别有5类:
['insufficient_site_enclosure', 'irregular_outgoing_wires_from_distribution_box', 'uncleaned_garbage', 'insufficient_energy_distribution_point']
但是completion_contents并没有完全遵守<answer>类别名</answer>的格式，而是直接输出类别名

{'loss': 0.0007, 'grad_norm': 1.7371084385095814, 'learning_rate': 3.448275862068965e-08, 'completion_length': 7.25, 'rewards/accuracy_reward': 0.5, 'rewards/format_reward': 0.0, 'reward': 0.5, 'reward_std': 0.0, 'kl': 0.01666259765625, 'epoch': 0.95}
 97%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████▏    | 28/29 [03:35<00:07,  7.57s/it]Invalidate trace cache @ step 0 and module 42282: cache has only 0 modules
completion_contents: ['uncleaned_garbage', 'inaugural_outgoing_wires_from_distribution_box', 'electricated_outgoing_wire', 'inferior_electrical_wires_submerged_in_water']
completion_contents: ['irregular_outgoing_wires_from_distribution_box', 'inadequate_site_enclosure', 'uncleaned_garbage', 'inaadequate_site_enclosure']
Invalidate trace cache @ step 0 and module 43011: cache has only 0 modules
completion_contents: ['inadequate_site_enclosure', 'irregular_outgoing_wires_from_distribution_box', 'incorrect', 'irregular_outgoing_wires_from_distribution_box']completion_contents: ['inadequate_site_enclosure', 'uncleaned_garbage', 'inadequate_site_enclosure', 'irregular_outgoing_wires_from_distribution_box']

{'loss': 0.0019, 'grad_norm': 31.79127511107813, 'learning_rate': 0.0, 'completion_length': 7.9375, 'rewards/accuracy_reward': 0.25, 'rewards/format_reward': 0.0, 'reward': 0.25, 'reward_std': 0.39433756470680237, 'kl': 0.0477294921875, 'epoch': 0.98}
{'train_runtime': 237.0005, 'train_samples_per_second': 0.494, 'train_steps_per_second': 0.122, 'train_loss': 0.0005493140931305568, 'epoch': 0.98}

**Comment by Liuziyu77** [comment]
可能是format_reward设计有问题，你可以给我看看你的format_reward和accuracy reward吗

**Comment by rxmao** [comment]
> 可能是format_reward设计有问题，你可以给我看看你的format_reward和accuracy reward吗

已经解决了，谢谢您！

**Comment by ZhaoyangLi-1** [comment]
最后怎么解决的
